docs/_static/vdr/v1.yaml

```diff
@@ -150,6 +150,89 @@ paths:
             text/plain:
               schema:
                 type: string
+  /internal/vdr/v1/did/{did}/verificationmethod:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example:
+          - "did:nuts:1234"
+        schema:
+          type: string
+    post:
+      summary: Creates a new verificationMethod and adds it to the DID document
+      operationId: addNewVerificationMethod
+      tags:
+        - DID
+      responses:
+        "201":
```

Comment:
a 201 links to the location where the newly created resource can be located. A 200 is more suitable